src/components/hint/index.tsx

```diff
@@ -549,40 +546,24 @@ class Hint extends Component<HintProps, HintState> {
   }
 
   render() {
-    const {onBackgroundPress, backdropColor, testID} = this.props;
-
     if (!this.props.visible && this.state.hintUnmounted) {
       return this.props.children || null;
     }
 
     return (
       <>
         {this.renderChildren()}
-        {onBackgroundPress ? (
-          <Modal
-            visible={this.showHint}
-            animationType={backdropColor ? 'fade' : 'none'}
-            overlayBackgroundColor={backdropColor}
```

Comment:
Notice that `backdropColor` was supported when using the Modal. 
We should make sure it is still supported 
Better check for other props as well

Reply:
Fixed, also added the animation support.